values.yaml

```diff
@@ -0,0 +1,22 @@
+global:
+  acme:
+    domain: steadops.local.steadforce.com
+  istio:
+    ingressGateway:
+      namespace: istio-ingress
+kiali:
+  resources:
+    limits:
```

Comment:
Why no resources on regular clusters?

Reply:
because it's quite new and we don't had sensible recommendations from vpa....